modules/zenith-public/sql/guild_crafter_point_npc_expansion_tags.sql

```diff
@@ -0,0 +1,32 @@
+-- Guild Point and Crafter Point NPC Expansion Tags
+-- Guild Point NPCs are tagged to COP (Chains of Promathia)
+-- Crafter Point NPCs are tagged to SOA (Seekers of Adoulin)
+-- Using calculated zoneid from npcid + name matching as npcid values can change with client updates
+-- Zone formula: (npcid - 16777216) >> 12
+
+-- Guild Point NPCs - Tag to COP (Chains of Promathia)
+UPDATE `npc_list` SET `content_tag` = 'COP' WHERE CONCAT((`npcid` - 16777216) >> 12, '-', `name`) IN (
```

Comment:
the operation used everywhere else in the codebase would be `>> 12) & 0xFFF` i.e. this should do what we want, in a way that is more understandable than the magic number `16777216`

```suggestion
UPDATE `npc_list` SET `content_tag` = 'COP' WHERE CONCAT((`npcid` >> 12) & 0xFFF, '-', `name`) IN (
```

Same for the second statement

Reply:
updated to remove magic numbers lol